rebirthdb/net.py

```diff
@@ -387,8 +400,10 @@ def close(self):
             try:
                 self._socket.shutdown(socket.SHUT_RDWR)
                 self._socket.close()
-            except Exception as ex:
+            except ReqlError as ex:
```

Comment:
What do you think about using `exc` instead of `ex`? As far as I know variable names should be at least three chars.

Reply:
Also, consistent with other refactoring, I missed this and will fix tomorrow.